File: AspNetMvcUnleashed/0672329980_source/Chapter06/CS/MvcApplication1/MvcApplication1/Controllers/BankController.cs
using System.Web.Mvc;

namespace MvcApplication1.Controllers
{
    public class BankController : Controller
    {
        //
        // GET: /Bank/Withdraw

        public ActionResult Withdraw()
        {
            return View();
        }

        //
        // POST: /Bank/Withdraw
        [AcceptVerbs(HttpVerbs.Post)]
        [ValidateAntiForgeryToken]
        public ActionResult Withdraw(decimal amount)
        {
            // Perform withdrawal
            return View();
        }

    }
}

File: AspNetMvcUnleashed/0672329980_source/Chapter06/VB/MvcApplication1/MvcApplication1/Controllers/BankController.vb
Public Class BankController
	Inherits Controller
	'
	' GET: /Bank/Withdraw

	Public Function Withdraw() As ActionResult
		Return View()
	End Function

	'
	' POST: /Bank/Withdraw
	<AcceptVerbs(HttpVerbs.Post), ValidateAntiForgeryToken> _
	Public Function Withdraw(ByVal amount As Decimal) As ActionResult
		' Perform withdrawal
		Return View()
	End Function

End Class
